src/Strategy/Node.hs

```diff
@@ -138,6 +146,12 @@ analyzeNpmLock (Manifest file) graph = do
 
 analyzeNpm :: (Has Diagnostics sig m) => PkgJsonGraph -> m DependencyResults
 analyzeNpm wsGraph = do
+  _ <-
+    recover $
+      warnOnErr MissingEdges
+        . warnOnErr MissingDeepDeps
+        $ errCtx (MissingNodeLockFile) $ fatalText "Lock files - yarn.lock or package-lock.json were not discovered."
```

Comment:
[nit]: Use `void`, use `.` and `$` consistently, formatting.
```suggestion
  void
    . recover 
    . warnOnErr MissingEdges
    . warnOnErr MissingDeepDeps
    . errCtx (MissingNodeLockFile)
    $ fatalText "Lock files - yarn.lock or package-lock.json were not discovered."
```

Reply:
fixed in https://github.com/fossas/fossa-cli/pull/805/commits/c64f7b9247ca7f0d676674640ce83ccdda5c2e34